libraries/AP_BattMonitor/AP_BattMonitor_FuelFlow.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "AP_BattMonitor.h"
+#include "AP_BattMonitor_Backend.h"
+
+class AP_BattMonitor_FuelFlow : public AP_BattMonitor_Backend
+{
+public:
+
+    /// Constructor
+    AP_BattMonitor_FuelFlow(AP_BattMonitor &mon, AP_BattMonitor::BattMonitor_State &mon_state, AP_BattMonitor_Params &params);
+
+    /// Read the battery voltage and current.  Should be called at 10hz
+    void read() override;
+
+    /// returns true if battery monitor provides consumed energy info
+    bool has_consumed_energy() const override { return has_current(); }
```

Comment:
Why not just return true here? This monitor can't change at runtime, it will always have current. (The compiler may or may not be picking this up, but it's less clear by calling the method anyways).

Reply:
fixed